tests/compiler/resolve-access.ts

```diff
@@ -0,0 +1,25 @@
+export function arrayAccess(): string {
+    let arr: u64[] = [1];
```

Comment:
indentation seems to differ from other tests here too

Reply:
Ah it's because my IDE settings. Will change to 2 spaces